docs/dev/api/insights.md

```diff
@@ -1002,20 +1002,101 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
 
 </details>
 
----
-
-## Activity
+## Filters
 
-### Get Activity
+### Get Filter Items
 
-<details><summary><span className="api get">GET</span> <code>/rest/v1/users/&#123;user_id&#125;/activity</code></summary>
+<details><summary><span className="api get">GET</span> <code>/v2/insights/&#123;source&#125;/filters</code></summary>
 <p/>
 
-Return daily statistics about test results and concurrency usage for the specified <code>user_id</code>.
+Return data for all tests that match the request criteria.
 
 #### Parameters
 
 <table id="table-api">
+  <tbody>
+    <tr>
+     <td><code>source</code></td>
+       <td><p><small>| PATH | REQUIRED | STRING |</small></p><p>Return results only for tests run in virtual device cloud or real device cloud. Supported values are:</p><p>
+     <ul>
+      <li><code>rdc</code> - Real Device Cloud</li>
+      <li><code>vdc</code> - Virtual Device Cloud</li>
+    </ul></p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>org_id</code></td>
+       <td><p><small>| QUERY| REQUIRED | STRING |</small></p><p>Return results only for the specified <code>org_id</code>.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>status</code></td>
+     <td><p><small>| QUERY | OPTIONAL | ARRAY of STRINGS |</small></p><p>Limit results to only those with a specified status. Supported values are:</p><p>
+     <ul>
+      <li><code>passed</code></li>
+      <li><code>error</code></li>
+      <li><code>failed</code></li>
+      <li><code>complete</code></li>
+    </ul></p>Default value is: <code>["error", "failed", "passed", "complete"]</code>
```

Comment:
@LoryApiFortress All values are default? The description of the same parameter for the Analytics > Get Test Results endpoint doesn't include a default.
<img width="777" alt="Screenshot 2023-06-05 at 5 06 06 PM" src="https://github.com/saucelabs/sauce-docs/assets/15352964/9bea5112-e460-4126-b65a-90152ee9749f">

Reply:
@lysannep yes, this is what the OpenAPI file says. I requested the review from Pawel and Hannah to double-check that all is correct.